src/client/controllers/movie.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react'
+
+const apiUrl = 'http://localhost:4000'
+
+const Movie = ({ token }) => {
+  const [movie, setMovie] = useState({
+    title: '',
+    description: '',
+    runtimeMins: null
+  })
+  const [movies, setMovies] = useState([])
+  const [movieResponse, setMovieResponse] = useState('')
+
+  const handleMovieChange = (e) => {
+    const { value, name } = e.target
+    const updatedValue = name === 'runtimeMins' ? parseInt(value) || '' : value
+    setMovie((prevMovie) => ({
+      ...prevMovie,
+      [name]: updatedValue
+    }))
+  }
```

Comment:
Bro, being honest this is just hard to read :joy: is there a way to separate these into a few more steps?